### PR TITLE
#73 73-popover-component

### PR DESCRIPTION
**Pull request by success-h**
added popover component to docs

### CODE REVIEW

File: content/docs/components/popover/usage.mdx [hunk 1]
```diff
@@ -0,0 +1,453 @@
+---
+title: Popover
+package: "@nature-ui/popover"
+description: Popover is a non-modal dialog that floats around a trigger
+---
+
+Popover is a non-modal dialog that floats around a trigger. It is used to
+display contextual information to the user, and should be paired with a
+clickable trigger element.
```
Comment (DNature) [inline content/docs/components/popover/usage.mdx:9]: This is a duplicate description because we already have `description` in the front-matter metadata.

Comment (success-h) [reply content/docs/components/popover/usage.mdx:9]: Thanks so much for the corrections. I just made the corrections

Comment (success-h) [reply content/docs/components/popover/usage.mdx]: please review again

File: content/docs/components/popover/usage.mdx [hunk 2]
```diff
@@ -0,0 +1,454 @@
...
+title: Popover
+package: "@nature-ui/popover"
+description: Popover is a non-modal dialog that floats around a trigger
+---
+
+It is used to display contextual information to the user, and should be paired with a
+clickable trigger element.
+
+`Popover` is built on top of the [Popper.js](https://popper.js.org/) library,
+and composes the `Popper` component.
```
Comment (DNature) [inline content/docs/components/popover/usage.mdx:13]: This is a continuation of the description. please move it there.

**Review by DNature — CHANGES_REQUESTED** [review]

File: content/docs/components/popover/usage.mdx (hunk 2 above)
Comment (DNature) [reply content/docs/components/popover/usage.mdx:13]: 9 - 14

**Review by DNature — APPROVED** [review]
LGTM

**Pull request merged**